keepAnimals

```diff
@@ -0,0 +1,7 @@
+def keepAnimals(animal_dic):
+  keepAnimals_dic = {}
+  for key, value in animal_dic.items():
+    if key in animal_list:
+      keepAnimals_dic[value] = key
```

Comment:
You inverted key and value. Instead you should do `keepAnimals_dict[key] = value`.